python/mdvtools/tests/test_stress_performance.py

```
@@ -0,0 +1,455 @@
+        
+        for n_cells, n_genes in sizes:
+            print(f"\nTesting size: {n_cells:,} cells x {n_genes:,} genes")
+            
+            # Create dataset
+            start_time = time.time()
+            adata = factory.create_massive_dataset(n_cells, n_genes)
+            creation_time = time.time() - start_time
+            
+            # Convert dataset
+            with temp_mdv_project() as test_dir:
+                with suppress_anndata_warnings():
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)